api_module.py

```diff
@@ -0,0 +1,289 @@
+        parse = [' '.join(name) for name in user_name]
+        full_names.append(parse)
+
+    return full_names
+
+
+@db_connection
+def get_bank_with_largest_capital(cursor):
+    """
+    Retrieve the bank name with the largest capital.
+
+    :param cursor: The database cursor object to execute SQL commands.
+    :return: A string indicating the bank that have the largest capital.
+    """
+    cursor.execute(SELECT_FROM_TABLE.format('Bank_id, Amount', 'Account'))
+    acc = cursor.fetchall()
+    banks = {bank_id: sum(user[1] for user in acc if user[0] == bank_id) for bank_id, _ in acc}
+
+    b_id = max(banks, key=banks.get)
+    cursor.execute(SELECT_BY_ID.format('name', 'Bank'), (b_id,))
+    bank = cursor.fetchone()[0]
+    return f'Bank which operates the biggest capital - {bank}'
+
+
+@db_connection
+def get_bank_with_oldest_client(cursor):
+    """
+    Retrieve the name of the bank that serves the oldest client.
+
+    :param cursor: The database cursor object to execute SQL commands.
+    :return: A string indicating the bank that serves the oldest client.
+    """
+    cursor.execute(SELECT_FROM_TABLE.format('Birth_day', 'User'))
+    oldest_client_birth_day = min(birth_day[0] for birth_day in cursor.fetchall())
+
+    cursor.execute('SELECT id FROM User WHERE Birth_day = ?', (oldest_client_birth_day,))
+    user_id = cursor.fetchone()[0]
+    bank_name = get_bank_name(user_id)
+    return f'Bank which serves the oldest client - {bank_name}'
+
+
+@db_connection
+def get_bank_with_highest_outbound_users(cursor):
+    """
+    Retrieve bank with the highest outbound users.
+
+    :param cursor: The database cursor object to execute SQL commands.
+    :return: A string indicating the bank that have the highest outbound users.
+    """
+    cursor.execute(SELECT_FROM_TABLE.format('Bank_sender_name, Account_sender_id', 'Transact'))
+    banks_users_count = {}
+
+    for bank_name, account_id in cursor.fetchall():
+        cursor.execute(SELECT_BY_ID.format('User_id', 'Account'), (account_id,))
+        user_id = cursor.fetchone()
+
+        banks_users_count.setdefault(bank_name, set())
+        banks_users_count[bank_name].add(user_id)
+
+    bank_with_highest_users = max(banks_users_count,
+                                  key=lambda bank: len(banks_users_count[bank]))
+
+    return f'Bank with the highest users which performed outbound transactions ' \
+           f'- {bank_with_highest_users}'
+
+
+def delete_users_or_accounts(cursor, table):
+    if table == 'User':
+        cursor.execute('SELECT id FROM User WHERE Name IS NULL OR Surname IS NULL '
+                       'OR Birth_day IS NULL')
+    else:
+        cursor.execute('SELECT id FROM Account WHERE User_id IS NULL OR Type IS NULL '
+                       'OR Account_Number IS NULL OR Bank_id IS NULL OR '
+                       'Currency IS NULL OR Amount IS NULL OR Status IS NULL')
+    return [row[0] for row in cursor.fetchall()]
```

Comment:
let's solve this task by python tools.
so, you need to fetch all data and then filter it with python